test/server/enrichment-map/enrichment/enrichment-test.js

```diff
@@ -1,72 +1,90 @@
 const chai = require('chai');
 const expect = chai.expect;
-const {enrichment} = require('../../../server/enrichment-map/enrichment');
+const _ = require('lodash');
+const {enrichment} = require('../../../../src/server/enrichment-map/enrichment');
+
+
+// object keys are unordered
+const objectEquality = (obj1, obj2) => {
```

Comment:
lodash should have some function that does this for you.

Reply:
probably see if something like this works:
https://lodash.com/docs/4.17.5#isEqual

Reply:
value for property 't group' is different every time. Ignore the property when checking equality.